docroot/modules/custom/foia_webform/src/Plugin/QueueWorker/FoiaSubmissionQueueWorker.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\QueueWorker;
+
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+use Drupal\webform\WebformSubmissionInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides base functionality for the FoiaForm Queue Workers.
+ */
+abstract class FoiaSubmissionQueueWorker extends QueueWorkerBase implements ContainerFactoryPluginInterface {
```

Comment:
Because we're only planning on having a single queue worker (we don't plan on leveraging Drupal's built in Cron system, so we won't have a manual vs cron worker) I'm not sure there is added value in implementing this as an abstract class. I'm thinking we can just have this as our concrete worker (i.e. let's add the `@QueueWorker` annotation here)